src/main/java/uk/gov/hmcts/reform/civil/config/SecurityConfiguration.java

```diff
@@ -44,7 +44,8 @@
         "/health/readiness",
         "/status/health",
         "/",
-        "/loggers/**"
+        "/loggers/**",
+        "/testing-support/**"
```

Comment:
Why need to add the "/testing-support/**" ?

Reply:
We don't, I have to remove that change